docs/enterprise/concepts.md

```diff
@@ -0,0 +1,140 @@
+
+#### Viewer
+
+A user with the Viewer role can:
+
+- view all resources in a Namespace (Routes, Policies, Certificates), including child Namespaces
+- view traffic dashboard for routes in the Namespace, including child Namespaces
+- view the activity log for a namespace.
+
+#### Manager
+
+In addition to the access provided by the Viewer role, the Manager can create, read, update, and delete routes, policies, and certificates in a Namespace (as wel as its children). A Manager may also reference policies and certificates in the parent Namespace.
+
+#### Admin
+
+An Admin user has permissions across all Namespaces. They can manage global settings, sessions, and service accounts, as well as view events and runtime data.
+
+## Users and Groups
+
+Pomerium populates users and groups from your IdP. This data is cached to prevent hitting API rate-limits, and provides look-ahead support when adding users or groups to [Namespaces](#namespaces) and [Policies](#policies).
+
+### Non-Domain Users
+
+When using Google as your IdP, you may hit an edge case where users are not available for lookup when the user's domain doesn't match that of your organization. In this case, there are two workarounds:
+
+- Create a group with the non-domain users in it. This group can be found and added to Namespaces and Policies.
+- Manually add the user's unique ID. Identify the ID from a user's Session Details page.
+
+   You can access the session details page from the 403 page, or by navigating to `/.pomerium` from any valid route. The unique ID is listed as "sub" under User Claims:
+
+   ![The Session Details page, showing the "sub" data](./img/session-details.png)
+
+## Service Accounts
+
+Service accounts provides bearer token based authentication for machine-to-machine communication through Pomerium to your protected endpoints. They can provide auth for monitoring services, create API integrations, etc.
+
+Service accounts can represent identities from your IdP or be Pomerium-only identities.
+
+## Routes
+
+Routes define the connection pathway and configuration from the internet to your internal service. As a very basic level, a route sends traffic from `external-address.company.com` to `internalService-address.localdomain`, restricted by the policies associated with it, and encrypted by your TLS certificates. But more advanced configurations allow identity header pass-through, path and prefix rewrites, request and response header modification, load balancer services, and more.
+
+### Protected Endpoints
+
+This term refers to the system or service the route provides or restricts access to.
+
+## Policies
+
+A Policy defines what permissions a set of users or groups has.
+
+Policies can be applied to [Routes](#routes) directly, or enforced within a [Namespace](#namespaces). This associates the set of permissions with a service or set of services, completing the authentication model.
```

Comment:
```
This associates the set of permissions with a service or set of services, completing the authorization model.
```

Not sure I follow.

Reply:
I think this was my take on your explanation of looking at the concepts as a whole. I'm trying to tie the concepts of policies, namespaces, and routes as part of the single model of authn/authz.

I'm happy to take any suggestions for rewriting that last sentence, or to just remove it.